drghs-worker/maintner-rtr/main.go

```diff
@@ -104,7 +111,27 @@ func main() {
 		log.Fatalf("failed to listen: %v", err)
 	}
 
-	grpcServer := grpc.NewServer(grpc.UnaryInterceptor(unaryInterceptorLog))
+	projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
+	exporter, err := texporter.NewExporter(texporter.WithProjectID(projectID))
+	if err != nil {
+		log.Fatalf("texporter.NewExporter: %v", err)
+	}
+
+	config := sdktrace.Config{DefaultSampler: sdktrace.ProbabilitySampler(0.01)}
+	tp, err := sdktrace.NewProvider(sdktrace.WithConfig(config), sdktrace.WithSyncer(exporter))
```

Comment:
These traces don't appear to be piggybacking off the trace coming from esp, is that possible?

Reply:
Not as far as I can tell. It appears as though the incoming headers from ESP are not using the standard OpenTelemetry headers so they are not being picked up. I verified this by checking the incoming request metadata (see commented out lines)